assets/js/utils/rtcUtils.ts

```diff
@@ -182,8 +175,8 @@ export const shareScreen = async (
 
     const newTrack = findTrackByType(client, "video", peersState);
     if (!newTrack) return peersState;
-    if (peersState.sourceIds["video"]) webrtc.replaceTrack(peersState.sourceIds["video"], newTrack);
-    else peersState.sourceIds["video"] = webrtc.addTrack(newTrack, peersState.peers[client.email].stream);
+    if (peersState.sourceIds["video"]) webrtcReplaceTrack(webrtc, newTrack, peersState, "video");
+    else peersState.sourceIds["video"] = webrtcAddTrack(webrtc, newTrack, peersState, client);
```

Comment:
This code looks very similar to the code changed below, maybe we should extract it to a function?